spyder/plugins/variableexplorer/widgets/tests/test_arrayeditor.py

```diff
@@ -29,6 +29,12 @@
 from spyder.plugins.variableexplorer.widgets.arrayeditor import ArrayEditor, ArrayModel
 
 
+# =============================================================================
+# Constants
+# =============================================================================
+FILES_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Please change the name of this constant to `HERE` instead of `FILES_PATH` because it points to the directory where this file is placed.